src/mongo_odm/model.hpp

```diff
@@ -71,6 +112,12 @@ class model {
      *
      * @warning This must be called with a new mongocxx::collection instance for every thread for
      *          the ODM to be thread-safe.
+     *
+     * @warning The mongocxx::client collection with which the passed-in mongocxx::collection was
```

Comment:
Maybe:
"The parent mongocxx::client from which the mongocxx::collection argument was created must outlive any of this model's CRUD methods. If the client object goes out of scope, a new collection must be passed to this method before using any CRUD methods."